fw/sock.c

```diff
@@ -820,6 +820,9 @@ ss_tcp_process_skb(struct sock *sk, struct sk_buff *skb, int *processed)
 	void *conn;
 	struct sk_buff *skb_head = NULL;
 	struct tcp_sock *tp = tcp_sk(sk);
+	int save_err_code = SS_OK, tmp_save_err_code;
```

Comment:
I must notice that `save_err_code` has life time the same `ss_tcp_process_skb()` it means, client can send multiple bad packets with some interval, but `save_err_code` will be `SS_OK`.

Reply:
In this case we use Conn_Stop bit. If error occurs on previus call of this function Conn_Stop bit was set.